### PR TITLE
HTML/CSS & JavaScript capstone project - WHO Conference

### PR DESCRIPTION
**Pull request by Alusp**
### IN THIS PR

- [link of the of video](https://www.loom.com/share/ad35d2ba5a344b1f92361b992a39826e)

- I used CSS for styling
- I build 2 pages:
    1. The home page.
    1. The about page.

- with screen sizes of:
    1. Mobile: up to 768px wide.
    1. Desktop: 768px or wider.

- gave it some Links for Interactions:
- The home page have a link in the menu to the about page.

- Mobile menu:
    1. When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
    2. gave it some styling (colors, typography, spacings, etc.).
    3. Gave it a close (X) button that closes the menu.

- Dynamic page:
    1. The section "Featured speakers"  created dynamically in JavaScript.
    2.  I gave it JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: pages/about.html [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+    </a>
+    <p class="logo-text">
+      2022 World health Assembly. All Rights Reserved.
+      This is registred under the MIT license. for more
+      info, please <a
+        href="https://github.com/Alusp/World-Health-Assembly">See
+        here</a>
+    </p>
+
+  </footer>
```
Comment (leonardodiasb) [inline pages/about.html:304]: - [ ] Your footer should have the logo aligned to the left and the text to the right both on desktop and mobile versions as it is in the template.
- [ ] It's really nice that you tried to follow the World Health Organization colors from their website but it makes this project color theme a little bit awkward. The provided template has a white background with dark text and logo for both pages on desktop and mobile versions and a dark background with white text for the About page desktop view. Kindly try to follow these rules on your project.
Template:
image
image

File: pages/about.html [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+                News
+              </a>
+            </li>
+          </ul>
+          <button type="button"
+            class="compaign-button button">
+            WHO Campaign
+          </button>
+        </div>
+      </nav>
```
Comment (leonardodiasb) [inline pages/about.html:170]: - [ ] The blue color from the World Health Organization is very different from the color theme from the template you used. A white background for the navbar is more appropriate. Kindly change that.
Template:
image

File: index.html [hunk 3]
```diff
@@ -122,177 +68,185 @@
...
-
-  <script src="script.js"></script>
-
-</body>
-
+    </header>
+
+    <main>
+      <section class="head">
+        <div class="head-bg"></div>
```
Comment (leonardodiasb) [inline index.html:143]: - [ ] The background you chose for the Home and About page `head` section are interesting but it makes it hard to read it's content. Try applying a lighter filter to improve the contrast with the dark text colors.

File: pages/about.html [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+      </li>
+    </ul>
+  </div>
+
+  <!-- absolutely positioned elements -->
+
+  <header>
+
+    <div class="mobile-nav">
+      <button class="mobile-menu-button" type="button">
```
Comment (leonardodiasb) [inline pages/about.html:99]: - [ ] Your mobile menu button is not working properly on your About page. Please, check what's happening and fix this issue.

File: index.html [hunk 5]
```diff
@@ -122,177 +68,185 @@
...
+            </div>
+
+            <p>
+              The Proceedings is to enable dialog and not constitute an verbatim
+              record of the proceedings.
+            </p>
+          </li>
+        </ul>
+        <a class="see-more">SEE THE WHOLE PROGRAM</a>
+      </section>
```
Comment (leonardodiasb) [inline index.html:216]: - [ ] The middle section of the Home page should contain 5 individual articles and they should be organized in a row for the desktop version. Your list only include 4 articles and they are not responsive to a row style in all desktop screen sizes. Kindly fix that.
Template:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Alusp, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

- [ ] Your video is great but it seems that you forgot to show a piece of code you are proud of making. Please submit another video no longer than 5 minutes and highlight a piece of code you find interesting.

- [ ] It seems that you forgot to acknowledge the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),  [Cindy Shin](https://www.behance.net/adagio07) on your README.

- [ ] You used the correct fonts provided by the template but it seems that you forgot to use the colors from the basic color swatch: #272A31, #EC5242, #D3D3D3. Please try to change the 'oranges', and 'grays' to match the colors from the template.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -4,6 +4,9 @@
 
 The World Health Assembly is the decision-making body of WHO. It is attended by delegations from all WHO Member States and focuses on a specific health agenda prepared by the Executive Board. The main functions of the World Health Assembly are to determine the policies of the Organization, appoint the Director-General, supervise financial policies, and review and approve the proposed programme budget. The Health Assembly is held annually in Geneva, Switzerland.
 
+
+![screenshot](images/Screenshot2.png)
+
```
Comment (ahmetbozaci) [inline README.md:9]: - [ ] Like the last code reviewer said, kindly add an acknowledgment section in the readme file that should include a design link. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.)

File: pages/about.html [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+    <footer class="license">
+      <a class="logo" href="#">
+        <img src="../images/h-logo-white.svg" alt="logo" />
+      </a>
+      <p class="logo-text">
+        2022 World health Assembly. All Rights Reserved. This is registred under
+        the MIT license. for more info, please
+        <a href="https://github.com/Alusp/World-Health-Assembly">See here</a>
+      </p>
+    </footer>
```
Comment (ahmetbozaci) [inline pages/about.html:246]: - [ ] Lİke the last code reviewer said kindly change the background color of the footer on the about and index page. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️ 


@Alusp  ✋🏼 
#### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
___ 

### 💯  Highlights 💯
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___

### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
⚠️ ⚠️  ⚠️ ⚠️  ⚠️ ⚠️  ⚠️ ⚠️ @Alusp 
- [ ] Your flow in the pull request is not correct. You should open pull request main to feature branch. Next time kindly check the project [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Make%20sure%20that%20you%20used%20correct%20GitHub%20flow.) detail.
- [ ] Nice to create another video but you can show another code that you are proud of it. I think showing the hover button style is not interesting. For example, you can show something challenging for you when creating a project.
⚠️ ⚠️  ⚠️ ⚠️  ⚠️ ⚠️  ⚠️ ⚠️ @Alusp 
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by AshakaE** [comment]
Hi @Alusp 🙋‍♂️, 

Thanks for making the requested changes 👏

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**